server/master/src/main/java/org/apache/accumulo/master/recovery/RecoveryManager.java

```diff
@@ -183,7 +183,6 @@ public boolean recoverLogs(KeyExtent extent, Collection<Collection<String>> walo
         String filename = master.getFileSystem().getFullPath(FileType.WAL, walog).toString();
         String dest =
             RecoveryPath.getRecoveryPath(master.getFileSystem(), new Path(filename)).toString();
-        log.debug("Recovering " + filename + " to " + dest);
```

Comment:
While this may not seem useful, it appears to be a statement that shows that the recovery manager is, at a minimum, doing something.  Maybe log progress higher up in the loop? (line 169?)

Reply:
This is logging what walogs each tablet has for every pass through the assignment loop, even if its not going to take any action.  That is why I was thinking it was useless, because its not based on taking an action.  Its just logging what was seen in the metadata table.

Reply:
For 1.9 its probably best to leave the log message as is in case anyone has automation that may be looking for it.  I will remove the log message in 2.1